gobblin-docker/gobblin-service/alpine-gaas-latest/entrypoint.sh

```diff
@@ -15,8 +15,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 GOBBLIN_HOME="$(cd `dirname $0`/..; pwd)"
+ARG_FLAG=""
+# Treat the config env args differently as they need to be combined together in 1 arg
+CONFIG_ARGS=""
+shopt -s nocasematch
+for i in "$@"
+do
+    case "$1" in
+        --args)
+            ARG_FLAG="--args"
+            CONFIG_ARGS="$2"
+            shift
+        ;;
+    esac
+    shift
+done
 
-./bin/gobblin.sh service gobblin-as-service start alpine-gaas-latest_gobblin-standalone
-busybox tail -F $GOBBLIN_HOME/logs/gobblin-as-service.out
+./bin/gobblin.sh service gobblin-as-service start ${ARG_FLAG} "${CONFIG_ARGS}" $@
+busybox tail -F $GOBBLIN_HOME/logs/gobblin-as-service.out $GOBBLIN_HOME/logs/gobblin-as-service.err
```

Comment:
does busybox becomes requirements/dependencies for GAAS ?

Reply:
It was to workaround a bug with tailing files in kubernetes, but busybox becomes unneeded in https://github.com/apache/incubator-gobblin/pulls
Regardless, this file is for executing GAAS in a dockerized context so the dependencies should be all within the container